### PR TITLE
refactor: 💡 [JIRA: IOSSDKBUG-402] use styleable tag templete

### PR DESCRIPTION
**Pull request by xiaoyu0722**
|light|dark|
|-|-|
|Simulator Screenshot - iPhone 16 Pro Max - 2024-10-28 at 17 17 26|Simulator Screenshot - iPhone 16 Pro Max - 2024-10-28 at 17 17 33|

### CODE REVIEW

File: Sources/FioriSwiftUICore/Utils/ConfigurationViewWrapper.swift [hunk 1]
```diff
@@ -8,6 +8,10 @@ public struct ConfigurationViewWrapper: View {
         self.v = view
     }
     
+    init(_ view: any View) {
+        self.v = view
+    }
```
Comment (billzhou0223) [inline Sources/FioriSwiftUICore/Utils/ConfigurationViewWrapper.swift:13]: Why do you add this init?

Comment (xiaoyu0722) [reply Sources/FioriSwiftUICore/Utils/ConfigurationViewWrapper.swift:13]: I'm also confused that compiler failed without this.
Screenshot 2024-10-29 at 07 22 16

Comment (xiaoyu0722) [reply Sources/FioriSwiftUICore/Utils/ConfigurationViewWrapper.swift:13]: I'm not sure if it's caused by system's api.
`nonisolated public func tag<V>(_ tag: V, includeOptional: Bool = true) -> some View where V : Hashable`

Comment (billzhou0223) [reply Sources/FioriSwiftUICore/Utils/ConfigurationViewWrapper.swift:13]: Not sure. I think we should keep only one of the init. If `any View` work fine, you can remove the other init.

**Review by billzhou0223 — APPROVED** [review]

**Pull request merged**